Project-1/README.md

```diff
@@ -13,63 +13,226 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
 
 1. Provide the truth tables for each of the following propositional logic formulas. State whether each is a tautology, a contradiction, or contingent:
   ```
-  (a) (¬A→B)∨((A∧¬C)→B) 
-  (b) (A→B)∧(A→¬B)
-  (c) (A→(B∨C))∨(C→¬A) 
-  (d) ((A→B)∧C)∨(A∧D) 
+  (a) (¬A→B)∨((A∧¬C)→B) Tautology
+  A	B	C	((¬A → B) ∨ ((A ∧ ¬C) → B))
+F	F	F	T
+F	F	T	T
+F	T	F	T
+F	T	T	T
+T	F	F	T
+T	F	T	T
+T	T	F	T
+T	T	T	T
+
+  (b) (A→B)∧(A→¬B) Contingent
+  A	B	((A → B) ∧ (A → ¬B))
+F	F	T
+F	T	T
+T	F	F
+T	T	F
+
+  (c) (A→(B∨C))∨(C→¬A) Tautology
+  A	B	C	((A → (B ∨ C)) ∨ (C → ¬A))
+F	F	F	T
+F	F	T	T
+F	T	F	T
+F	T	T	T
+T	F	F	T
+T	F	T	T
+T	T	F	T
+T	T	T	T
+  (d) ((A→B)∧C)∨(A∧D) Contingent
+  A	B	C	D	(((A → B) ∧ C) ∨ (A ∧ D))
+F	F	F	F	F
+F	F	F	T	F
+F	F	T	F	T
+F	F	T	T	T
+F	T	F	F	F
+F	T	F	T	F
+F	T	T	F	T
+F	T	T	T	T
+T	F	F	F	F
+T	F	F	T	T
+T	F	T	F	F
+T	F	T	T	T
+T	T	F	F	F
+T	T	F	T	T
+T	T	T	F	T
+T	T	T	T	T
   ```
 	
 2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
   ```
   (a) (A→B)→C
+  (A∨C)∧(¬B∨C)
   (b) (A→(B∨C))∨(C→¬A)
+  Av~A
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
+ Bv~B
   ```
   
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   
   ```
-  (a)  B(x,y) that says that x is a brother of y  
-  (b)  A(x,y) that says that x is an aunt of y  
-  (c)  C(x,y) that says that x and y are cousins   
-  (d)  O(x) that says that x is an only child  
-  (e)  T(x) that says that x has exactly two brothers 
+  Note that any definitions require that V has access to the connectives and quantifiers of first-order logic, which is not specified in the prompt. I have assumed that they are available, as otherwise there is no way to define any of the formulas in the following examples. 
```

Comment:
It is specified in the prompt. That is what is meant by "Let V be the vocabulary of first-order logic..." The clause that follows is a specification of a binary and unary relation within that vocabulary.

Reply:
It might have been better to say "fully specified" or "explicitly" to convey my meaning there. I meant only to note that the precise connectives and quantifiers and such were not explicitly noted so I would use whichever best suited the definitions in question. This was due to potential issues where we would usually only have access to the specific resources explicitly mentioned in the question in similar problems in Logic, as I initially took these questions to involve a language only with the relations you explicitly mentioned before I changed it to include the relevant connectives.

In either case, I removed that clause to get around this issue.